src/schemas/epicPayload.schema.json

```diff
@@ -1,144 +1 @@
-{
```

Comment:
I'm not sure what this is about, but maybe it's time to remove these?

Reply:
It's also broken the tests

Reply:
I'm fine with removing the old schema/validation stuff in this PR if you want to.
This was for the server-side validation of POST requests, but it has not been used in PROD.

Server-side validation of client requests is actually less important, because we don't store data or reveal information as a result of these requests. The worst that can happen is someone doesn't get an epic.

Whereas client-side validation of props prevents users from seeing broken components on the site.

Reply:
Yeah that makes sense. I've taken it out. Now we're using `zod` for the client side validation stuff, it wouldn't be too tricky to use it for server side validation too